### PR TITLE
Fix typo in document

### PR DESCRIPTION
**Pull request by yeojin-dev**
Fix typo

### CODE REVIEW

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

**Review by ikhoon — APPROVED** [review]
Nice catch!

File: site/src/pages/tutorials/grpc/blog/index.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import versions from '/gen-src/versions.json';
 
 Learn to build a gRPC service with Armeria by writing a blog service in gRPC. This tutorial is based on a [sample service](https://github.com/line/armeria-examples/tree/main/tutorials/grpc), a minimal blog service, with which you can create, read, update, and delete blog posts.
 
-Before we get our hands on the tutorial, let's switfly go over Armeria's gRPC features. Armeria supports features that the upstream (gRPC-Java) does not support.
+Before we get our hands on the tutorial, let's swiftly go over Armeria's gRPC features. Armeria supports features that the upstream (gRPC-Java) does not support.
```
Comment (minwoox) [inline site/src/pages/tutorials/grpc/blog/index.mdx:12]: This will be fixed by https://github.com/line/armeria/pull/5445/files so let's revert this. 😉

Comment (minwoox) [inline site/src/pages/tutorials/grpc/blog/index.mdx:12]:
```suggestion
Before we get our hands on the tutorial, let's switfly go over Armeria's gRPC features. Armeria supports features that the upstream (gRPC-Java) does not support.
```

File: site/src/pages/tutorials/grpc/blog/index.mdx [hunk 2]
```diff
@@ -84,4 +84,3 @@ dependencies {
   runtimeOnly "ch.qos.logback:logback-classic:${versions['ch.qos.logback:logback-classic']}"
 }
 `}</CodeBlock>
```
Comment (minwoox) [inline site/src/pages/tutorials/grpc/blog/index.mdx:86]:
```suggestion
`}</CodeBlock>

```

**Review by minwoox — APPROVED** [review]
Thanks a lot!
I push the revert commit for the index.mdx file so that #5445 isn't conflict. 😉

**Pull request merged**